components/contribution-flow/StepDetails.js

```diff
@@ -126,8 +127,31 @@ const StepDetails = ({ onChange, data, collective, tier, showPlatformTip, router
                 currencyDisplay="full"
                 prependProps={{ color: 'black.500' }}
                 required
+                onKeyDown={event => {
+                  setPressedKey(event.key);
+                }}
```

Comment:
Seems a bit hacky to rely on the state for this, could we rather look at the event directly in `onChange`?